packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.tsx

```diff
@@ -427,14 +427,14 @@ export class KlevuQuicksearch {
       // No results page settings
       if (this.tPopularProductsTitleOnNoResultsPage === undefined) {
         this.tPopularProductsTitleOnNoResultsPage =
-          settings?.klevu_uc_userOptions.noResultsOptions.productsHeading ??
+          settings?.klevu_uc_userOptions.noResultsOptions?.productsHeading ??
           getTranslation("quicksearch.tPopularProductsTitle")
       }
       if (this.hideTrendingProductsOnNoResultsPage === undefined) {
-        this.hideTrendingProductsOnNoResultsPage = !settings?.klevu_uc_userOptions.noResultsOptions.showPopularProducts
+        this.hideTrendingProductsOnNoResultsPage = !settings?.klevu_uc_userOptions.noResultsOptions?.showPopularProducts
```

Comment:
to make it consistent we can access `settings?.klevu_uc_userOptions` with optional operator everywhere

Reply:
issue is not with settings?.klevu_uc_userOptions, this will always be defined, its the next level that can be undefined, we noticed for priceFormatter, added here for all